src/configuration/services/saas.md

```diff
@@ -4,9 +4,22 @@ title: Magento Services
 
 Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Services]({% link configuration/services.md %}) >  Magento Services
 
-Magento allows you to link your Magento store to a SaaS service through the Admin UI. By configuring and managing SaaS Environments, you can connect to services such as Product Recommendations.
+Magento allows you to link your Magento store to a SaaS service through the Admin UI and an API key. By configuring and managing SaaS Environments, you can connect to services such as Product Recommendations. To begin, you must generate an API key.
 
-When you create a [Magento account]({% link magento/magento-account.md %}), identified by a MageID, you can generate a Magento API key to connect to the SaaS services within Magento. To connect to Magento Commerce SaaS services, the API key needs to be generated by the license-holder. If you are a Magento partner, the contract signer needs to generate the API key. Then the API key can be passed to the systems integrator or development team, who can then manage the SaaS projects and environments on behalf of the license-holder.
+## Generate an API key {#apikeys}
+
+When you create a [Magento account]({% link magento/magento-account.md %}), identified by a MageID, you can generate a Magento API key to connect to the SaaS services within Magento. The API key needs to be generated by the license-holder. If you are a Magento partner, the contract signer needs to generate the API key. Then, the API key can be passed to the systems integrator or development team, who can manage the SaaS projects and environments on behalf of the license-holder.
```

Comment:
To utilize Magento Commerce SaaS services, the API key needs to be generated by the license-holder.